lib/slop/options.rb

```diff
@@ -26,7 +26,7 @@ class Options
     def initialize(**config)
       @options    = []
       @separators = []
-      @banner     = "usage: #{$0} [options]"
+      @banner     = config[:banner] ? config[:banner] : "usage: #{$0} [options]"
```

Comment:
Do you think this should support passing `false` in to remove a banner altogether? if so this condition wouldn't work, otherwise it could probably be `config.fetch(:banner, "usage: #{$0} [options]")`